### PR TITLE
RE: [Edit] C++ Add Codebytes #2156

### LINKED ISSUE #2156: [Edit] C++ Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.       
https://www.codecademy.com/resources/docs/cpp/math-functions/cbrt
https://www.codecademy.com/resources/docs/cpp/math-functions/ceil
https://www.codecademy.com/resources/docs/cpp/math-functions/cos

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by xGavi**
### Description

Closes #2156
Hey!
I have updated cos.md, ceil.md and cbrt.md to include a Codebyte example. I have also slightly altered the wording on the previously given examples to better reflect the content of the page. This should close issue #2156.
Regarding this being my first contribution, I came across this repo through the Learn Git & GitHub course on Codecademy, and did this contribution as part of one of its projects.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/cos/cos.md
Comment (dakshdeepHERE) [inline content/cpp/concepts/math-functions/terms/cos/cos.md:56]:
```suggestion
```

**Comment by xGavi** [comment]
> Few changes

Fixed!

File: content/cpp/concepts/math-functions/terms/cbrt/cbrt.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cbrt/cbrt.md:43]: Can we make this codbyte a little more different than the prior example?

File: content/cpp/concepts/math-functions/terms/ceil/ceil.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/ceil/ceil.md:43]: Can we make the codebyte example a little more different than the prior example.

File: content/cpp/concepts/math-functions/terms/cos/cos.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cos/cos.md:41]: Can we make the codebyte a little more different than the prior example?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@xGavi I did a first maintainer review. The main issue is that the codebytes are all copied from the prior examples with just a number being changed. The codebytes should have different code than the prior examples in the entry.

**Comment by xGavi** [comment]
@SSwiniarski all examples were changed!

**Review by SSwiniarski — APPROVED** [review]
@xGavi looks good for a second review.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@xGavi I've added some changes to this mainly with transferring the output into `shell` blocks since the outputs are slightly lengthy to be kept in in-line comments. When transferring to `shell` blocks, please place a sentence before each `shell` saying something like "The example outputs the following:" or "The following is the output:". You can see an example of this in the [acosh entry](https://www.codecademy.com/resources/docs/cpp/math-functions/acosh) in the Example section.

**Review by KTom101 — APPROVED** [review]
@xGavi I've made a few more minor changes directly and ran the command `npm run format:all` in terminal to fix the failed [ci checks](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=continuous%20integration%20(CI)%20workflows). Looks good to go 👍

**Comment by KTom101** [comment]
@xGavi Edits are live: [cbrt()](https://www.codecademy.com/resources/docs/cpp/math-functions/cbrt), [ceil()](https://www.codecademy.com/resources/docs/cpp/math-functions/ceil), [cos()](https://www.codecademy.com/resources/docs/cpp/math-functions/cos)

Screenshot 2023-07-31 at 7 52 34 PM

**Pull request merged**
